public/components/stub-modal/stub-modal.js

```diff
@@ -134,26 +138,34 @@ function StubModalInstanceCtrl($rootScope,$scope, $modalInstance, $window, confi
         );
     };
 
-    $scope.ok = function (addToComposer) {
-        var stub = $scope.stub;
-        var promise;
-
-        if (!addToComposer && !($scope.mode === "import")) {
-            delete stub.status;
-        }
-
-        if (addToComposer) {
-            promise = wfContentService.createInComposer(stub);
-        } else if (stub.id) {
-            promise = wfContentService.updateStub(stub);
+    $scope.ok = function (addToComposer, addToAtomEditor) {
+        const stub = $scope.stub;
+        let promise;
```

Comment:
minor: this could be its own function to avoid the `let`, e.g:

```js
function foo() {
  if ($scope.contentName === 'Atom') {
    return wfContentService.createInMediaAtomMaker(stub);
  } else {
    return wfContentService.createStub(stub);
  }
}
```